scale/job/job_type_serializers.py

```diff
@@ -40,7 +40,7 @@ class JobTypeBaseSerializerV6(ModelIdSerializer):
 
 class JobTypeListSerializerV6(JobTypeBaseSerializerV6):
     """Converts job type model fields to REST output"""
-    id = None
+    id = serializers.IntegerField()
```

Comment:
You should probably just delete the line for consistency and it'll inherit the id from the model serializer.